### ISSUE TITLE
Detect the used Network Interface on first run

### ISSUE THREAD

**Issue by XhmikosR** [XhmikosR/perfmonbar#3]
Detect the Network Interface on the first run.

**Comment by super-lurker** [comment]
I know I am 5 years late, but I am also having the same issue, how did you fix it?

**Comment by XhmikosR** [comment]
`[R]` should work.

**Comment by super-lurker** [comment]
fixed, thankss :)

**Comment by XhmikosR** [comment]
I've yet to find a complete list of special characters and their
replacements

**Comment by s-rog** [comment]
`Network Interface(*)` works for me

**Comment by XhmikosR** [comment]
> 
> 
> `Network Interface(*)` works for me

Yeah, but this includes all network interfaces which isn't quite the same. There's overhead etc. But it's a solution albeit not a perfect one :P
